app/models/package.rb

```diff
@@ -539,6 +543,14 @@ def dispatched_orders_package
     orders_packages.get_dispatched_records_with_order_id(order_id).first
   end
 
+  def get_designated_orders_packages
```

Comment:
can these just be scopes on orders_packages?

Not a huge fan of the word 'get' - feels very much like Java and not really necessary.

Would be good to see:

    orders_packages.designated

Same re dispached below